site/content/docs/developer-guide/plugin-manifest.md

```diff
@@ -0,0 +1,198 @@
+    ...
+```
+
+#### Specifying platform-specific instructions
+
+Krew makes it possible to install the same plugin on different operating systems
+(like `windows`, `darwin` (macOS), and `linux`) and different architectures
+(like `amd64`, `386`, `arm`).
+
+To support multiple platforms, you may need to define multiple `platforms` in
+the plugin manifest. The `selector` field matches to operating systems and
+architectures using the keys `os` and `arch` respectively.
+
+**Example:** Match to a Linux or macOS platform, any architecture:
+
+```yaml
+...
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: "os", operator: "In", values: [darwin, linux]}
+```
+
+**Example:** Match to Windows 64-bit:
+
+```yaml
+  platforms:
+  - selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+```
+
+**Example:** Match to Linux (any architecture):
+
+```yaml
+  platforms:
+  - selector:
+      matchLabels:
+        os: linux
+```
+
+The possible values for `os` and `arch`  come from the Go runtime. Run
+`go tool dist list` to see all possible platforms and architectures.
+
+#### Specifying files to install
+
+Each operating system may require a different set of files from the archive to
+be installed.
+
+You can use the `files` field to specify
+which files should be copied into the plugin directory while extracting the
+files out of archive.
+
+The `files:` list specifies the copy operations (like `mv(1) <from> <to>`) to
+the files `from` the archive `to` the installation destination.
+
+* **Example:** Extract all files (default behavior, you can omit `files:` altogether)
+
+> If the `files:` list is unspecified, it defaults to:
+> ```yaml
+> files:
+> - from: *
+>   to: .
+> ```
+
+* **Example:** Copy specific files:
+
+  ```yaml
+  files:
+  - from: LICENSE
+    to: .
+  - from: bin/foo-windows.exe
+    to: foo.exe
+  ```
+
+* **Example:** Use wildcards to match multiple files:
+
+  ```yaml
+  files:
+  - from: "*/*.sh"
+    to: "."
+  ```
+
+  As a result of this operation, the copied out files will preserve their
+  directory structure in the extracted directory.
+
+#### Specifying plugin executable
+
+Each `platform` field requires a path to the plugin executable in the plugin's
+installation directory.
+
+For example, if your plugin executable is named `foo.sh`, specify:
+
+```yaml
+platforms:
+  - bin: "./foo.sh"
+    ...
+```
+
+Krew will create a [symbolic link](https://en.wikipedia.org/wiki/Symbolic_link)
+named `kubectl-foo` (and `kubectl-foo.exe` on Windows) to your plugin executable
+after installation is complete. The name of the symbolic link comes from the
+plugin name (i.e. the `metadata.name` field).
+
+> **Note on underscore conversion:** If your plugin name contains dashes, krew
+> will automatically convert them to underscores for kubectl to be able to find
+> your plugin.
+>
+> For example, if your  is named `view-logs` and your plugin binary is named
```

Comment:
```suggestion
> For example, if your plugin name is `view-logs` and your plugin binary is named
```
I wonder if talking about the plugin binary only creates confusion. If you want to make the point that the installed binary may have an arbitrary name because an additional link is created, you should be more explicit about it.